src/components/PaymentInformation/CardName.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { getCreditCardType } from '../../utils/creditCardType';
+import SectionHeader from '../SectionHeader';
+
+function CardName({ cardNumber }) {
+  const renderCardName = () => {
```

Comment:
I would change it a bit a little.
Create (and test it afterwards) separate function above component:
```
function creditCardName(cardNumber) {
  const cardType = creditCardType(cardNumber);

  switch(cardType) {
    case 'VISA':
       return 'Visa';
        ...
    default:
        return 'Unknown Credit Card';
    } 
}
```
and in component:
```
function CardName({ cardNumber }) {
  return <SectionHeader>{creditCardName(cardNumber)}</SectionHeader>
}
```